docs/application/native/guides/app-management/dali-widget-app.md

```diff
@@ -0,0 +1,269 @@
+# DALi Widget Application
+
+
+You can create a widget application with DALi to display small version of application on the Home screen.
+
+To create a DALi widget application, you must:
+
+- Use the [Dali::WidgetApplication class](#widget-application-api).
+
+	> **Note**
+	> - The implementation of DALi widget application is similar to [DALi basic UI application](dali-basic-app.md), because the [Dali::WidgetApplication](../../api/mobile/latest/classDali_1_1WidgetApplication.html) class inherits from the [Dali::Application](../../api/mobile/latest/classDali_1_1Application.html) class.
+  > - Ensure you are familiar with the basic UI application details before tackling a widget application.
+
+- Ensure you start the event loop.
+
+- To implement your own widget class inherit the `Dali::Widget` class.
+
+![DALi widget application](./media/dali_widget_application.png)
+
+**Figure: DALi widget application**
+
+For more information on basics of creating a DALi widget application, see [Create a DALi Widget Application](#create).
+
+<a name="widget-application-api"></a>
+## Widget Application Class
+
+If you want to use DALi for your widget application, you must create the application with the `Dali::WidgetApplication` class. This class provides the means for initializing the resources required by DALi.
+
+The table list the main functions to manage widget instance:
+
+**Table: Main functions**
+
+| Function                             | Description                              |
+|--------------------------------------|------------------------------------------|
+| `mainloop()`                         | This function is used to The mainloop() function is used to start the event loop. If you do not call the function and start the event loop, DALi cannot call any callback functions for application events.  |
```

Comment:
May there was a mistake in copy & paste. For now the sentence is "This function is used to The mainloop() function is used to start the event loop."

Reply:
@mijin85cho done